src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.awt.datatransfer.StringSelection;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Duke {
+    static void printHello() {
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
+    }
+    static void printTaskList(int index, Task[] Tasklist) {
+        for (int i = 0; i < index; i++) {
+            System.out.println((i + 1) +".  " + "["+ Tasklist[i].getStatusIcon() +"]\t" + Tasklist[i].description + "\n");
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] Tasklist = {};
```

Comment:
Tasklist can be initialized and assigned a value in the same line Task[] Tasklist = Task[100]